server/models/Board.js

```diff
@@ -0,0 +1,52 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const Column = require('./Column');
+
+const BoardSchema = new Schema({
+	name: {
+		type: String,
+		required: true
+	},
+	columns: [Column.schema]
+}, {collection: 'Board'});
+
+// creates a new instance of the Board model and returns the saved instance
+BoardSchema.statics.createNewBoard = async function(name) {
+	const inProgressCol = new Column({
+		name: 'In Progress'
+	});
+	await inProgressCol.save();
```

Comment:
Can you figure out a way to increase efficiency by making these promises all perform in parallel?